packages/p2p/src/stores/general-store.js

```diff
@@ -44,6 +44,7 @@ export default class GeneralStore extends BaseStore {
     is_listed = false;
     is_loading = false;
     is_p2p_blocked_for_pa = false;
+    is_p2p_user = null;
```

Comment:
have to add another observable for this, cause we can't use hooks inside the store 😢 but for v2 all of this will be gone 😄